R/plots_plan.R

```diff
@@ -19,6 +19,18 @@ plot_plan <- drake_plan(
     facet_wrap(~trait, scales = "free_y"),
   
 
+  #H1Q2 + 3: Conv/div along gradient and across treatments
+  Site <- c(H = "High Alpine",
```

Comment:
use = not <- in a plan

Reply:
I think I changed this in the next commit. will check